src/ListView/__mocks__/MockListManager.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import update from 'immutability-helper'
+import { mockListItems } from './mockListItems'
+import ListView from '../ListView'
+import ListViewItem from '../ListViewItem'
+import ListViewItemContainer from '../ListViewItemContainer'
+import ListViewItemExpansion from '../ListViewItemExpansion'
+
+class MockListManager extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      items: mockListItems
+    }
+    this.itemClicked = (e, i) => {
+      this.setState(prevState =>
+        update(prevState, {
+          items: {
+            [i]: {
+              active: {
+                $set: !prevState.items[i].active
+              }
+            }
+          }
+        })
+      )
+    }
+    this.childClicked = (e, i, j) => {
+      this.setState(prevState =>
```

Comment:
i wonder if we can provide this functionality as a default, and allow user to override it if he needs to...